app/src/main/java/org/mozilla/firefox/firefoxlitedownloader/utils/Utils.java

```diff
@@ -0,0 +1,35 @@
+package org.mozilla.firefox.firefoxlitedownloader.utils;
+
+import android.content.Context;
+import android.os.Environment;
+import android.util.Log;
+
+import java.util.Locale;
+
+
+public final class Utils {
+
+    private Utils() {
+        // no instance
+    }
+
+    public static String getRootDirPath(Context context) {
+        if (Environment.MEDIA_MOUNTED.equals(Environment.getExternalStorageState())) {
```

Comment:
Do you need permission here?
If there a pre-required before calling this method?